com.unity.render-pipelines.core/Runtime/Textures/RTHandleSystem.cs

```diff
@@ -181,10 +181,11 @@ public void SetReferenceSize(int width, int height, bool reset)
                 }
                 m_FramesSinceLastReset++;
             }
-
-            // If some cameras is requesting the same res as the max res, we don't want to reset
-            if (m_MaxWidths == width && m_MaxHeights == height)
```

Comment:
This is never hit if you have one target with a max height and another camera with a max width.
This case would case a forever feedback loop and reset forever. I propose we just be more conservative and always reset the counter. We only downsize if all of our views are significantly smaller than the current max allocated.